CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # OHHTTPStubs — CHANGELOG
 
+* Migrate samples to Swift 3
```

Comment:
Could you add a full stop and 2 spaces at the end of that line so that the markdown formatting matches the other notes?